suite-native/accounts/src/utils.ts

```diff
@@ -0,0 +1,77 @@
+import { A, D, G } from '@mobily/ts-belt';
+
+import { AccountType, networks } from '@suite-common/wallet-config';
+import { formattedBitcoinAccountTypeMap } from '@suite-common/wallet-core/src/accounts/constants';
+import { Account } from '@suite-common/wallet-types';
+import { getNetwork } from '@suite-common/wallet-utils';
+
+const accountTypeToSectionHeader: Readonly<Partial<Record<AccountType, string>>> = {
+    normal: 'default',
+    taproot: 'Taproot',
+    segwit: 'Legacy Segwit',
+    legacy: 'Legacy',
+    ledger: 'Ledger',
+};
+
+/**
+ * Returns true if account label, network name, account type or account included token contains filter value as a substring.
```

Comment:
You can use jsdoc `@description` then this description will be showed as tooltip which is nice

Reply:
IMO the `@description` tag is redundant. Content of any `/** ... */` above a definition is displayed in the tooltip.

![Screenshot 2024-01-04 at 12 40 07](https://github.com/trezor/trezor-suite/assets/26143964/4f940dca-f43a-4135-baf6-8a2ffc988881)

Reply:
I would still prefer to use @description because it works only because it vscode, but whatever